SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts

```diff
@@ -155,11 +169,14 @@ export class PixiTileMapHelper {
       } else if (layer instanceof EditableTileMapLayer) {
         const tileLayer = layer as EditableTileMapLayer;
 
+        const tileWidth = tileLayer.tileMap.getTileWidth();
+        const tileHeight = tileLayer.tileMap.getTileHeight();
+
         for (let y = 0; y < tileLayer.tileMap.getDimensionY(); y++) {
```

Comment:
should the dimensionY and dimensionX be calculated outside the loops too? Or can they change?

Reply:
I could move them.